### ISSUE TITLE
(URGENT) Single "items" that are not numbered are now numbered and cannot resolve

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-bipm#52]
`relaton fetch "CIPM Resolution (1879)"` used to work until today I found it no longer works.

```
$ bundle exec relaton fetch "CIPM Resolution (1879)"
[relaton-bipm] ("CIPM Resolution (1879)") fetching...
[relaton-bipm] ("CIPM Resolution (1879)") not found.
No matching bibliographic entry found

$ bundle exec relaton fetch "CIPM Resolution 1 (1879)"
[relaton-bipm] ("CIPM Resolution 1 (1879)") fetching...
[relaton-bipm] ("CIPM Resolution 1 (1879)") found CIPM RES 1 (1879)
<bibdata type="proceedings" schema-version="v1.2.3">
  <fetched>2023-05-09</fetched>
```

Moreover, the content is now wrong -- these items should not be numbered if they were not originally numbered:

```
  <docidentifier type="BIPM" primary="true">CIPM RES 1 (1879)</docidentifier>
  <docidentifier type="BIPM" primary="true" language="en" script="Latn">CIPM RES 1 (1879, E)</docidentifier>
  <docidentifier type="BIPM" primary="true" language="fr" script="Latn">CIPM RES 1 (1879, F)</docidentifier>
  <docidentifier type="BIPM-long" language="en" script="Latn">CIPM Resolution 1 (1879)</docidentifier>
  <docidentifier type="BIPM-long" language="fr" script="Latn">Résolution 1 du CIPM (1879)</docidentifier>
  <docidentifier type="BIPM-long">CIPM Resolution 1 (1879) / Résolution 1 du CIPM (1879)</docidentifier>
  <docnumber>CIPM RES 1 (1879)</docnumber>
```

This should just be `CIPM RES 1 (1879)` and `CIPM Resolution (1879) / Résolution du CIPM (1879)`.

**Comment by ronaldtse** [comment]
Previously, this worked (`CGPM Resolution 1889-00`). Now it requires `CGPM Resolution 1889-01`, which is wrong.

**Comment by andrew2net** [comment]
Fixed in v1.14.7
```sh
$ relaton fetch "CIPM Resolution (1879)"
[relaton-bipm] ("CIPM Resolution (1879)") fetching...
[relaton-bipm] ("CIPM Resolution (1879)") found CIPM RES (1879)
<bibdata type="proceedings" schema-version="v1.2.3">
  <fetched>2023-05-09</fetched>
  <title format="text/plain" language="en" script="Latn">Signes abréviatifs pour les poids et mesures métriques</title>
  ...
  <docidentifier type="BIPM" primary="true">CIPM RES (1879)</docidentifier>
  <docidentifier type="BIPM" primary="true" language="en" script="Latn">CIPM RES (1879, E)</docidentifier>
  <docidentifier type="BIPM" primary="true" language="fr" script="Latn">CIPM RES (1879, F)</docidentifier>
  <docidentifier type="BIPM-long" language="en" script="Latn">CIPM Resolution (1879)</docidentifier>
  <docidentifier type="BIPM-long" language="fr" script="Latn">Résolution du CIPM (1879)</docidentifier>
  <docidentifier type="BIPM-long">CIPM Resolution (1879) / Résolution du CIPM (1879)</docidentifier>
  ...

$ relaton fetch "CIPM Resolution 1 (1879)"
[relaton-bipm] ("CIPM Resolution 1 (1879)") fetching...
[relaton-bipm] ("CIPM Resolution 1 (1879)") found CIPM RES (1879)
<bibdata type="proceedings" schema-version="v1.2.3">
  <fetched>2023-05-09</fetched>
  <title format="text/plain" language="en" script="Latn">Signes abréviatifs pour les poids et mesures métriques</title>
  ...

$ relaton fetch "CIPM RES 1 (1879)"
[relaton-bipm] ("CIPM RES 1 (1879)") fetching...
[relaton-bipm] ("CIPM RES 1 (1879)") found CIPM RES (1879)
<bibdata type="proceedings" schema-version="v1.2.3">
 ...

$ relaton fetch "CGPM Resolution 1889-00"
[relaton-bipm] ("CGPM Resolution 1889-00") fetching...
[relaton-bipm] ("CGPM Resolution 1889-00") found CGPM RES (1889)
<bibdata type="proceedings" schema-version="v1.2.3">
  <fetched>2023-05-09</fetched>
  <title format="text/plain" language="en" script="Latn">Sanction of the international prototypes of the metre and the kilogram</title>
 ...
```

**Comment by ronaldtse** [comment]
Confirm fixed, thank you @andrew2net !
